cmd/bootstrap/utils.go

```diff
@@ -68,8 +68,8 @@ func lookup(svcName string) (sets.Set[string], error) {
 	return endpoints, nil
 }
 
-func lockExists() (bool, error) {
-	return fileExists("/var/lib/mysql/bootstrap.lock")
+func lockExists(lockName string) (bool, error) {
```

Comment:
Looks like we only have `bootstrap` lock, do we need this `lockName` param at all?

Reply:
I was adding locks during development and I think this is more clear, I intend to keep it